### DISCUSSION TITLE
Mac M1 binary?

### DISCUSSION THREAD

**Discussion by scpedicini** [4ian/GDevelop#5087] [Archive] [answered]
When downloading from the official GDevelop website, the file provided is:
`GDevelop-5-5.1.158.dmg`

The release folder here has a arm64-mac version (presumably for Apple Silicon machines):
`GDevelop-5-5.1.158-arm64-mac.zip`

I downloaded the latter considering I have a Mac M1 Pro. Is the official site's dmg file a universal binary or will it run with Rosetta?

**Comment by 4ian** [comment, marked as answer]
You should be able to get the Apple Silicon version with this button: 
<img width="200" alt="image" src="https://user-images.githubusercontent.com/1280130/224086477-e7796f80-d2fa-4b58-a7ae-b85beda4efa8.png">

Probably would be better with a universal binary but for now it works well like this and we would need to find a contributor who is able to rework the Electron building process to make a universal binary :)

**Reply by scpedicini** [reply]
Wow. I *completely* missed that icon. Derp. Thanks!
